User Localization settings (#13181)
OrchardCMS/OrchardCore: src/docs/releases/1.9.0.md

Reviewer: Also add a paragraph of docs about this to the Users module's reference.

Added new extensions to make registering custom deployment step easier

- services.AddDeployment<TSource, TStep>();
- services.AddDeployment<TSource, TStep, TDisplayDriver>();
- services.AddDeploymentWithoutSource<TStep, TDisplayDriver>();

## Users Module

> Added new localization settings that allow the user to set his/her default culture and time zone.
> 
> !!! note
>     The newly added settings override the current admin localization culture and time zone settings.

Reviewer: ```suggestion
Added new localization settings in the User Localization feature that allow the user to set their default culture and time zone.

!!! note
    The newly added settings override the current admin localization culture and time zone settings if the User Localization feature is enabled.
```

Reviewer: Feel free to commit these directly and let the conversations auto-resolve themselves.

    }
}
```

## Users Module

> Added new user localization feature that allows to set default culture per user in the admin UI.

Reviewer: ```suggestion
Added a new User Localization feature that allows to be able to configure the culture per user from the admin UI.
```
